BAKKA_Editor/Views/MainView.axaml.cs

```diff
@@ -2715,18 +2825,26 @@ private void VisualBeatDivisionNumeric_OnValueChanged(object? sender, NumericUpD
 
         var value = (float) (e.NewValue ?? _vm.VisualBeatDivisionNumeric);
         if (value >= (float) _vm.VisualBeatDivisionNumericMinimum && value <= (float) _vm.VisualBeatDivisionNumericMaximum)
+        {
             // update
+            userSettings.ViewSettings.BeatDivision = value;
             skCircleView.BeatDivision = value;
+        }
         else
+        {
             // revert
-            _vm.VisualBeatDivisionNumeric = (decimal) skCircleView.BeatDivision;
+            _vm.VisualBeatDivisionNumeric = (decimal)skCircleView.BeatDivision;
+        }
     }
 
     private void GuideLine_OnSelectionChanged(object? sender, SelectionChangedEventArgs e)
     {
         if (skCircleView != null && sender != null)
         {
-            skCircleView.GuideLineSelection = ((ComboBox) sender).SelectedIndex;
+            var value = ((ComboBox)sender).SelectedIndex;
+
+            userSettings.ViewSettings.GuideLineSelection = value;
+            skCircleView.GuideLineSelection = value;
```

Comment:
OK for now, in the future we should stop duplicating state everywhere

Reply:
I tried just using `default` but the compiler didn't like that.

Reply:
Specifically I'd rather just give SkCircleView access to settings in some form, so that we can avoid this dance.